domain/src/main/java/life/qbic/domain/usermanagement/policies/FullNamePolicy.java

```diff
@@ -0,0 +1,38 @@
+package life.qbic.domain.usermanagement.policies;
+
+/**
+ * <b>Full Name Policy</b>
+ *
+ * <p>Validates if a String is concurrent with the full name requirements
+ *
+ * @since 1.0.0
+ */
+public class FullNamePolicy {
+
+  private static final int MIN_LENGTH = 1;
+
```

Comment:
Why did you add that? AFAIK we dont have a policy there other than that it needs to be provided

Reply:
I think a policy is necessary here since the Domain logic should decide on the validity of the input. 
Also this allows us to test our requirements easily and adapt the policy if necessary down the line. 
We could e.g. think about not allowing nonsensical names down the line as suggested e.g. here: https://support.google.com/a/answer/9193374?hl=en

Reply:
As I said above I dont see it as a policy. Its a check for presence, the other things are requirements you just came up with. Its nothing that we have for now. Therfore I would kindly ask you to remove the policy and check the value in the user entity (I think this was done like this before, right?). You can also do it in the use case but a policy is overkill for this

Reply:
I don't see the problem with having an easily extendable policy based solution that checks the user input. 
IMO the policy should contain ALL requirements for the associated user field.
Before the introduction of the policy we did not check the user input at all in the user entity.